packages/kit/test/apps/basics/src/routes/cookies/encoding/+page.svelte

```diff
@@ -0,0 +1,12 @@
+<script>
+
+	/** @type {import('./$types').PageData} */
+	export let data;
+
+
+</script>
+
+Cookie: <span id="cookie-value">
+  {data.a}
+</span>
```

Comment:
it's a little funny to me the start and end tags don't line up. i'd either put the start tag on a new line or just put everything on a single line
```suggestion
Cookie: <span id="cookie-value">{data.a}</span>
```